internal/controllers/gateway/gateway_controller.go

```diff
@@ -292,7 +292,7 @@ func (r *GatewayReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			return ctrl.Result{}, err
 		}
 		debug(log, gateway, "ensured object was removed from the data-plane (if ever present)")
```

Comment:
The log here is also necessary if `DeleteObject()` is removed.

Reply:
Hm 🤔 but it says "ensured object was removed" not "going to ensure" or "ensuring". It comments on what **has** happened, not what's going to happen.

Reply:
Sorry, I missed the tense.